### ISSUE TITLE
Showing in Latest Sales price for what was sold item?

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#7975]
In the past we had some price which item was sold, can we maybe have this again on item? 

Seems shows only once a time 

![image](https://github.com/kodadot/nft-gallery/assets/5887929/f39a2654-17f5-4096-9897-2d7beede8f35)

**Comment by roiLeo** [comment]
could be related with kodadot/stick#93

**Comment by daiagi** [comment]
duplicate of #8109?

**Comment by yangwao** [comment]
> duplicate of #8109?

yup
